packages/element/src/platform.js

```diff
@@ -0,0 +1,9 @@
+const Platform = {
```

Comment:
Since this seems like a copy of https://github.com/necolas/react-native-web/blob/0e81c6ef2758d4ca9b2099b1d04a4f8c417f0f43/packages/react-native-web/src/exports/Platform/index.js let's try to keep the original copyright attribution here.

Reply:
👍